### DISCUSSION TITLE
[question] 请问是否是代码内interface IPageSchema就是页面协议的定义？

### DISCUSSION THREAD

**Discussion by liaodalin19903** [opentiny/tiny-engine#877] [Q&A]
页面协议：
官方文档说明：
https://opentiny.design/tiny-engine#/protocol

请问：
1、是否是在这里，代码进行的协议定义（IPageSchema）呢？
https://github.com/opentiny/tiny-engine/blob/12bfefc821d435bf69e7e02bf288b5fed633588f/packages/vue-generator/src/index.d.ts#L102
 
2、但是发现官方文档的协议字段和代码的协议（IPageSchema）的字段是对不上的，是指的代码变更之后，文档没有及时更新是吗？

**Comment by chilingling** [comment]
1. 代码里 ts 定义的协议，用于出码转换，与完整的页面协议有一定的裁剪。
2. 官方文档的部分协议字段确实没有更新完整。

所以建议以数据库字段为最新的完整协议字段。

**Reply by liaodalin19903** [reply]
谢谢您的回答，那么`数据库字段`是指的搭建好tiny-engine之后，可以进行核验表结构没错吧。
